README.rst

```diff
@@ -0,0 +1,121 @@
+|Icon| |title|_
+===============
+
+.. |title| replace:: diffpy.nmf_mapping
+.. _title: https://diffpy.github.io/diffpy.nmf_mapping
+
+.. |Icon| image:: https://avatars.githubusercontent.com/diffpy
+        :target: https://diffpy.github.io/diffpy.nmf_mapping
+        :height: 100px
+
+|PyPi| |Forge| |PythonVersion| |PR|
+
+|CI| |Codecov| |Black| |Tracking|
+
+.. |Black| image:: https://img.shields.io/badge/code_style-black-black
+        :target: https://github.com/psf/black
+
+.. |CI| image:: https://github.com/diffpy/diffpy.nmf_mapping/actions/workflows/main.yml/badge.svg
+        :target: https://github.com/diffpy/diffpy.nmf_mapping/actions/workflows/main.yml
+
+.. |Codecov| image:: https://codecov.io/gh/diffpy/diffpy.nmf_mapping/branch/main/graph/badge.svg
+        :target: https://codecov.io/gh/diffpy/diffpy.nmf_mapping
+
+.. |Forge| image:: https://img.shields.io/conda/vn/conda-forge/diffpy.nmf_mapping
+        :target: https://anaconda.org/conda-forge/diffpy.nmf_mapping
+
+.. |PR| image:: https://img.shields.io/badge/PR-Welcome-29ab47ff
+
+.. |PyPi| image:: https://img.shields.io/pypi/v/diffpy.nmf_mapping
+        :target: https://pypi.org/project/diffpy.nmf_mapping/
+
+.. |PythonVersion| image:: https://img.shields.io/pypi/pyversions/diffpy.nmf_mapping
+        :target: https://pypi.org/project/diffpy.nmf_mapping/
+
+.. |Tracking| image:: https://img.shields.io/badge/issue_tracking-github-blue
+        :target: https://github.com/diffpy/diffpy.nmf_mapping/issues
+
+Run NMF analysis on PDF and XRD data
+
+* LONGER DESCRIPTION HERE
+
+For more information about the diffpy.nmf_mapping library, please consult our `online documentation <https://diffpy.github.io/diffpy.nmf_mapping>`_.
+
+Citation
+--------
+
+If you use diffpy.nmf_mapping in a scientific publication, we would like you to cite this package as
+
+        diffpy.nmf_mapping Package, https://github.com/diffpy/diffpy.nmf_mapping
```

Comment:
I think there is a paper/papers that need citing for this one.  Also in the docs

Reply:
<img width="982" alt="Screenshot 2024-08-11 at 10 54 02 AM" src="https://github.com/user-attachments/assets/1781c462-adee-4cea-9a11-2ada7d224989">

Fixed in `REAMDE.rst`. I will add this in the following PR where I move from `docs` to `doc` using `git mv`. @sbillinge

Reply:
Citation source: https://pdfitc.org/NMF

Reply:
I see the citation format, in diffpy.structure and diffpy.pdffit2. I will make a change.

<img width="701" alt="Screenshot 2024-08-11 at 11 04 40 AM" src="https://github.com/user-attachments/assets/b7b27a98-7360-48be-b7f4-af728e9481b6">

<img width="708" alt="Screenshot 2024-08-11 at 11 04 48 AM" src="https://github.com/user-attachments/assets/5ca95864-34df-45e3-b14e-c4620738083b">